tests/dataSources_test.go

```diff
@@ -852,39 +852,45 @@ var _ = Describe("DataSources", func() {
 				}, shortTimeout, time.Second).Should(HaveKeyWithValue(cdiLabel, cronName))
 			})
 
-			It("[test_id:8112] should restore DataSource if DataImportCron removed from SSP CR", func() {
-				// Wait until DataImportCron imports PVC and changes data source
-				Eventually(func() bool {
-					cron := &cdiv1beta1.DataImportCron{}
-					Expect(apiClient.Get(ctx, dataImportCron.GetKey(), cron)).To(Succeed())
+			Context("restore dataSource", func() {
+				var cron *cdiv1beta1.DataImportCron
+				JustAfterEach(func() {
+					logObject(client.ObjectKeyFromObject(cron), &cdiv1beta1.DataImportCron{})
```

Comment:
The `cron` temporary variable is nil at this point. You can get the key directly from `dataImportCron.GetKey()`:
```go
logObject(dataImportCron.GetKey(), &cdiv1beta1.DataImportCron{})
```